packages/hash/frontend/src/pages/[account-slug]/index.page.tsx

```diff
@@ -67,8 +67,12 @@ export const AccountHome: NextPage = () => {
       <Typography>
         Please select a page from the list, or create a new page.
       </Typography>
-    </MainContentWrapper>
+    </>
   );
 };
 
-export default AccountHome;
+Page.getLayout = function getLayout(page: ReactElement) {
+  return <LayoutWithSidebar>{page}</LayoutWithSidebar>;
+};
```

Comment:
```suggestion
Page.getLayout = getLayoutWithSidebar;
```
(+ change in imports)

Reply:
Thanks for catching. Updated here => bd28f865103bd5a5